### DISCUSSION TITLE
Toggle Boolean findOneAndUpdate

### DISCUSSION THREAD

**Discussion by moomoolive** [Automattic/mongoose#10099] [Q&A] [answered]
Hey guys,

Is it possible to toggle a boolean with "findOneAndUpdate"?

Here's an example of what I mean:
`users.findOneAndUpdate({ _id: user._id }, { isCool: !isCool })`

**Comment by vkarpov15** [comment, marked as answer]
Not atomically, I think your best bet would be to make `isCool` a number and use bitwise operators: https://docs.mongodb.com/manual/reference/operator/update/bit/#mongodb-update-up.-bit if you need to atomically toggle a boolean.

**Reply by moomoolive** [reply]
That's a great idea - thanks for the update
